components/global/Grid1x2.vue

```diff
@@ -54,22 +53,25 @@ export default {
 
 <style scoped>
 .feature-btn {
-  color: var(--color-gold) !important;
-  font-size: 1.1rem;
-  max-width: 200px;
+  background-color: #3273dc;
+  border-color: transparent;
+  text-decoration: none;
+  color: white;
+  font-size: 1.3rem;
   text-align: left;
+  border-radius: 8px;
 }
 
 .feature-btn:hover {
-  text-decoration: underline;
+  background-color: #378dff;
 }
 
 .feature-img {
-  width: 275px;
+  width: 100%;
+  border-radius: 16px;
 }
 
 .desc {
-  color: var(--color-text);
   font-weight: 300;
   line-height: 1.75rem;
   opacity: 0.9;
```

Comment:
Maybe 24 or 20px would be a better font size. The description looks like it's too big